server/src/main/java/org/elasticsearch/common/unit/Fuzziness.java

```diff
@@ -231,11 +231,13 @@ public boolean equals(Object obj) {
             return false;
         }
         Fuzziness other = (Fuzziness) obj;
-        return Objects.equals(fuzziness, other.fuzziness);
+        return Objects.equals(fuzziness, other.fuzziness) &&
+                Objects.equals(lowDistance, other.lowDistance) &&
+                Objects.equals(highDistance, other.highDistance);
```

Comment:
since `lowDistance` and `highDistance` are integers, should we just compare them with `lowDistance == other.lowDistance`?

Reply:
Good point, I guess I'm just lazy and prefer the reading repeated "Objects.equals... &&" but didn't care too much about the Autoboxing involved here. Will change.